pydrive2/files.py

```diff
@@ -518,6 +518,42 @@ def Delete(self, param=None):
         """
         self._FilesDelete(param=param)
 
+    @LoadAuth
+    def Copy(self, target_folder=None, new_title=None, param=None):
+        """Creates a copy of this file. Folders cannot be copied.
+
+        :param target_folder: Folder where the file will be copied.
+        :type target_folder: GoogleDriveFile
```

Comment:
consistency, if keep `,optional` below, let's do the same here please, if not - let's remove everywhere.